package.json

```diff
@@ -51,6 +51,8 @@
     "@types/async-retry": "1.4.9",
     "@types/body-parser": "1.19.6",
     "@types/cors": "2.8.19",
+    "@types/express": "^4.17.13",
```

Comment:
Needed because the integration test suite uses Express to set up a non-GraphQL server